secrets-gradle-plugin/src/test/kotlin/com/google/android/libraries/mapsplatform/secrets_gradle_plugin/SecretsPluginTest.kt

```diff
@@ -157,14 +156,18 @@ class SecretsPluginTest {
 
     private fun check(vararg keyValues: Pair<String, String>) {
         keyValues.forEach { (key, value) ->
-            Assert.assertEquals(value, placeholders[key])
-            verify(variant).buildConfigField("String", key, value.addParenthesisIfNeeded())
```

Comment:
This assertion step looks wrong. I guess you want to assert that the key-paired buildConfigField exists?